getline.ts/package.json

```diff
@@ -31,7 +31,8 @@
     "doc": "typedoc -out doc/ src typings node_modules/web3-typescript-typings/index.d.ts",
     "generate": "protoc --plugin=protoc-gen-ts=./node_modules/.bin/protoc-gen-ts --js_out=import_style=commonjs,binary:src/generated --ts_out=service=true:src/generated -I ../pb/ ../pb/*.proto",
     "tsc": "tsc",
-    "test": "mocha -r ts-node/register test/**/*.ts"
+    "test": "mocha -r ts-node/register test/**/*.ts",
+    "lint": "tslint -c tslint.json 'src/**/*.ts' -e src/generated/**/*"
```

Comment:
TSLint disables some checks if it gets all the files manually (`src/**/*.ts`) instead of through the project file.
Specifically type checking is disabled without the `--project` flag.
https://palantir.github.io/tslint/usage/cli/

Reply:
FYI - Typescript compiler `tsc` also disables type checking without the `--project` flag

Reply:
Good to know, will take a look later.